src/cmd/apply.ts

```diff
@@ -93,6 +90,7 @@ const applyAll = async () => {
     { streams: { stdout: debug.stream.log, stderr: debug.stream.error } },
   )
   await prepareValues()
+  debug.info('Deploying charts with label stage!=prep')
```

Comment:
```suggestion
  debug.info('Deploying charts with --label stage!=prep')
```

Reply:
it does not seem to improve readabillity